mscore/albummanager.cpp

```diff
@@ -60,6 +60,14 @@ AlbumManager::AlbumManager(QWidget* parent)
       createScore->setEnabled(false);
 
       MuseScore::restoreGeometry(this);
+          
+      if (album == 0) {
+          createNewClicked();
+          }
+          
+      if (albumName->text().isEmpty()) {
+          albumName->setEnabled(true);
+          }
```

Comment:
We use 6 spaces tab. See https://musescore.org/en/developers-handbook/musescore-coding-rules#Indentation-%26-Whitespaces